public/schema/JobTemplate.yaml

```diff
@@ -4,11 +4,11 @@ description: openQA job template
 type: object
 additionalProperties: false
 required:
-- architectures
 - products
+- scenarios
 properties:
-  architectures:
-    description: A set of architectures with a list of test suite(s) eg. ppc64le
+  scenarios:
+    description: The scenarios to run ie. lists of test suites per arch and medium
```

Comment:
I think "ie." is not the correct style, please write:
"The scenarios to run, i.e. lists of test suites per architecture and medium" with the comma and the additional dot and the spelled out "architecture"